front/src/components/UI/Form/DeleteModal.tsx

```diff
@@ -0,0 +1,42 @@
+import { Button, Group, Modal } from "@mantine/core";
+import { UserForAdmin as User } from '@app/models/types/User.ts';
+
+interface Props {
+  opened: boolean
+  close: () => void
+  user: User
+  onSubmit: () => void
+}
+
+export default function DeleteModal({
+  opened,
+  close,
+  user,
+  onSubmit,
+}: Props){
+
+  return (
+    <>
+      <Modal.Root opened={opened} onClose={close}>
+        <Modal.Overlay />
+        <Modal.Content>
+          <Modal.Header>
+            <Modal.Title>ATTENTION Vous êtes bien sûr de vouloir <strong>SUPPRIMER {user.email}</strong> ?</Modal.Title>
```

Comment:
```suggestion
            <Modal.Title>ATTENTION ! Êtes vous sûr de vouloir <strong>SUPPRIMER {user.email}</strong> ?</Modal.Title>
```
Isn't it like that ?

Reply:
J'ai plutôt corrigé avec translation 👍